### PR TITLE
Display name and port in SourceIndicator tooltip; clean up Tooltip styles

### PR DESCRIPTION
**Pull request by jaredscheib**
- [x] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #2056 
Connect #2068

### The Problem
- The SourceIndicator did not show what port its host was connected to.
- If a long source `host:name` existed, the string would be too wide for the Tooltip
- Sometimes the tooltip would hover in a non-down direction and its caret would be in the wrong place

### The Solution
- Make SourceIndicator Tooltip conform to `name @ host:port` convention for full insight
- Make Tooltip styles consistent between Graph Tips and SourceIndicator
- Make Tooltip not clip when near edge of screen
- Fix the directionality + caret issue

### CODE REVIEW

**Comment by alexpaxton** [comment]
screen shot 2017-10-04 at 12 40 07 pm
screen shot 2017-10-04 at 12 37 02 pm
screen shot 2017-10-04 at 12 36 48 pm
screen shot 2017-10-04 at 12 36 43 pm

No clipping!

File: ui/src/dashboards/constants/index.js [hunk 1]
```diff
@@ -104,5 +104,5 @@ export const DISPLAY_OPTIONS = {
 
 export const TOOLTIP_CONTENT = {
   FORMAT:
-    '<p>K/M/B = Thousand / Million / Billion</p><p>K/M/G = Kilo / Mega / Giga </p>',
+    '<p><strong>K/M/B</strong> = Thousand / Million / Billion<br/><strong>K/M/G</strong> = Kilo / Mega / Giga </p>',
 }
```
Comment (121watts) [inline ui/src/dashboards/constants/index.js]: Nice.  This will look way better.

File: ui/src/kapacitor/components/RuleHeaderEdit.js [hunk 2]
```diff
@@ -24,18 +24,16 @@ const RuleHeaderEdit = ({
           className="page-header__title page-header--editable kapacitor-theme"
           onClick={onToggleEdit}
           data-for="rename-kapacitor-tooltip"
-          data-tip="Click to Rename"
+          data-tip={'<p>Click to Rename</p>'}
         >
```
Comment (121watts) [inline ui/src/kapacitor/components/RuleHeaderEdit.js]: This is a string so the `data-tip` prop curly braces should be unecessary

Comment (alexpaxton) [reply ui/src/kapacitor/components/RuleHeaderEdit.js]: Done!

**Pull request merged**